tools/roslaunch/src/roslaunch/launch.py

```diff
@@ -540,7 +540,7 @@ def launch_node(self, node, core=False):
         except roslaunch.node_args.NodeParamsException as e:
             self.logger.error(e)
             if node.package == 'rosout' and node.type == 'rosout':
-                printerrlog("\n\n\nERROR: rosout is not built. Please run 'rosmake rosout'\n\n\n")
+                printerrlog("\n\n\nERROR: rosout is not built. Please run 'catkin_make rosout'\n\n\n")
```

Comment:
It seems to me that this message is addressed at a developer building ROS core libraries. I would think the official way to build ROS core libraries (and the only supported way) is `catkin_make_isolated` ?
Ref : http://wiki.ros.org/Installation/Source